src/NetworkListener.h

```diff
@@ -0,0 +1,90 @@
+/** \file NetworkListener.h
+ *  @brief The interface used by the core logic to get notified of incoming requests.
```

Comment:
This file does not belong in this PR but it stays for now (as interface reference)